app/redux/node/reducer.ts

```diff
@@ -1,23 +1,27 @@
 import type { NodeState, CustomAction } from '../../types';
 import { LOGOUT } from '../auth/actions';
-import { SET_NODE_STATUS, SET_NODE_ERROR, SET_NODE_VERSION_AND_BUILD } from './actions';
+import { SET_NODE_ERROR, SET_NODE_STATUS, SET_NODE_VERSION_AND_BUILD } from './actions';
 
 const initialState = {
   status: null,
   version: '',
   build: '',
   port: '',
-  error: 0
+  error: null
 };
 
 const reducer = (state: NodeState = initialState, action: CustomAction) => {
   switch (action.type) {
     case SET_NODE_STATUS: {
-      const { status } = action.payload;
-      return { ...state, status };
+      const status = action.payload;
+      return {
+        ...state,
+        status,
+        error: null
```

Comment:
this creates a possibility to overwrite existing "error" object to null

Reply:
Yep. But if we are getting the status from the Node it means that Node is working. Meanwhile, we are notifying the User only about critical errors (Node does not respond) we don't have to store errors, which might be actual while we are getting status updates.
So this is not a bug, this is a feature to tell the User "Ok. Now there are no errors" ;)